src/content/blog/2025_fosdem.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Quantum Computing at FOSDEM 2025
+author: Purva Thakre
+day: 10
+month: 1
+year: 2025
+tags: 
+  - Community
+---
+
+**After five years, the Quantum Computing devroom returns to FOSDEM.**
+
+FOSDEM (Free and Open Source Software Developers' European Meeting) is a conference scheduled for 1-2 February 2025 at the ULB (Université libre de Bruxelles) Solboch Campus in Brussels, Belgium. The Quantum Computing devroom is slated for the second half of 2 February 2025.
+
+<p align="center">
+  <a href="https://fosdem.org/2025/schedule/"> 
+    <img src="/images/fosdem_25.png" />
+  </a>
+</p>
+
+
+FOSDEM serves as a venue for open-source developers around the world to meet and facilitate conversations about the open-source ecosystem. This year, we hope to foster connections between the classical and quantum open-source ecosystems. Talks and presentations in each devroom (developer room) are suited to a particular topic of interest. In late 2024, we [put out a call](https://unitary.fund/community/FOSDEM_2025/) for the quantum devroom on talks related to: quantum software, quantum compilation, quantum programming languages, securing quantum communications, etc. 
+
```

Comment:
I propose to remove this section all together. This blog post is more a call for attendance, we want to be short and direct. The call for participation was a story for last year :)